services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -180,6 +187,34 @@ async def cached_registry_services() -> dict[str, Any]:
     return [s for s in services_details if s is not None]
 
 
+class _ServicesInaccessibleBody(BaseModel):
+    services_to_check: list[ServiceKeyVersion]
+
+
+@router.post(
+    "/inaccessible",
```

Comment:
her as well ```/inaccessible``` is not a resource.
I think I would go with ```:shareable``` as well.
Also I would put everything in the body

Reply:
Ok I have a little bit redo the endpoints they are more restful :+1: let me know what you think. 
https://github.com/ITISFoundation/osparc-simcore/pull/4253#issue-1715533321
(For now, I preferred the naming inaccessible before the shareable in the catalog, but we can discuss)